cli.js

```diff
@@ -133,21 +145,69 @@ function startProgressLogging (stats) {
   })
 }
 
-function printScanProgress (stats) {
+function printScanProgress (stats, last) {
   var dirCount = stats.directories + 1 // parent folder
-  logger.stdout(
-    'Creating share link for ' + stats.files + ' files, ' +
-    dirCount + ' folders,' +
-    (stats.size ? ' ' + prettyBytes(stats.size) + ' total' : '')
+  var msg = chalk.bold.blue('Calculating Size: ')
+  if (last) msg = chalk.bold.green('Creating New Dat: ')
+  msg += chalk.bgBlue.white(
+    '[ ' + stats.files + ' files | ' + dirCount + ' folders | ' +
+    (stats.size ? ' ' + prettyBytes(stats.size) + ' total' : '') + ' ]'
   )
+  logger.stdout(msg)
 }
 
 function printAddProgress (statsAdd, statsScan) {
-  logger.stdout(
-    'Fingerprinting file contents (' + statsAdd.filesRead +
-    '/' + statsScan.files + ')' +
-    ' ' + Math.floor(100 * (statsAdd.bytesRead / statsScan.size)) + '%'
+  var msg = ''
+  var indent = '  ' // indent for single file info
+  var i = 0
+
+  while (statsAdd.files.length > 1) {
```

Comment:
This line I am confused about. I tried `statsAdd.files.length > 0` but sometimes it'd end up inside the while loop but with an empty array. I think it was b/c the in progress files but couldn't quite grasp it.